src/main/java/org/jenkinsci/test/acceptance/plugins/credentials/Domain.java

```diff
@@ -0,0 +1,35 @@
+package org.jenkinsci.test.acceptance.plugins.credentials;
+
+import org.jenkinsci.test.acceptance.po.Control;
+import org.jenkinsci.test.acceptance.po.Describable;
+import org.jenkinsci.test.acceptance.po.PageArea;
+import org.jenkinsci.test.acceptance.po.PageAreaImpl;
+import org.jenkinsci.test.acceptance.po.PageObject;
+
+@Describable("Credential Domain")
+public class Domain extends PageAreaImpl {
+
+    public final Control name = control("/domain/name");
+    public final Control description = control("/domain/description");
+    public final Control addCredentialButton = control("/hetero-list-add[credentials]");
+    public final Control addSpecificationButton = control("/domain/hetero-list-add[specifications]");
+    
+    public Domain(PageObject context, String path) {
+        super(context, path);
+    }
+    
+    public Domain(PageArea area, String relativePath) {
```

Comment:
Sounds rights :) Will fix it once I get some more input from you in the other comments in case we thing there are more things to improve/add. Cheers